_config.yaml

```
@@ -35,7 +35,11 @@ header_pages:
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
+  - workshop/posting-errors-guidelines.md
+  - working-with-your-data/working-with-your-own-data.md
```

Trying to think about whether we should adjust the order here. Also, I think we can probably get away with not having the software setup in the nav bar. People hopefully only need that once, and it is linked from the home page.
```suggestion
  - workshop/workshop-materials.md
  - workshop/resources-for-consultation-sessions.md
  - working-with-your-data/working-with-your-own-data.md
  - workshop/posting-errors-guidelines.md
```